core/src/main/java/com/adobe/aem/commons/assetshare/content/properties/impl/AssetTypeImpl.java

```diff
@@ -143,6 +149,10 @@ protected void activate(Cfg cfg) {
         )
         String audioLabel() default AUDIO_LABEL;
 
+        @AttributeDefinition(
```

Comment:
I like this approach of using a computed property to make the determination "is a CF". However the configurable label is an issue here. If a user changes the label in the osgi config everything related to Content Fragments breaks.
![image](https://user-images.githubusercontent.com/8974514/40267747-411ff9da-5b30-11e8-9b75-b38a54c77986.png)

Originally the AssetType class was intended as a "display" of the high level mimetype and was not intended to do any logic based switching. The configurable labels is a nice touch but maybe its not necessary (i18n should take care of localization use case). I'm wondering if we simply do away with the configurable labels? Or we should introduce a 2nd computed property to differentiate between Type and Display Type. 

wdyt @davidjgonzalez , @arpithaar ?

Reply:
I don't love the mixing of `adaptTo` language being called as a computed property. seems dirty?
+1 for `contentFragment` -> returning a boolean.